pages/unsubscribe.js

```diff
@@ -0,0 +1,238 @@
+import Head from "next/head";
+import { useState } from "react";
+import Joi from "joi";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { ErrorBox } from "../components/molecules/ErrorBox";
+import { useRouter } from "next/router";
+import { useTranslation } from "next-i18next";
+import { Layout } from "../components/organisms/Layout";
+import { ActionButton } from "../components/atoms/ActionButton";
+import { TextField } from "../components/atoms/TextField";
+
+export default function Unsubscribe(props) {
+  const { t } = useTranslation("common");
+  const { asPath, push } = useRouter();
+
+  // Joi form validation schema. Only required fields are validated
+  const formSchema = Joi.object({
+    email: Joi.string()
+      .email({ tlds: { allow: false } })
+      .required()
+      .error((errors) => {
+        errors.forEach((error) => {
+          switch (error.code) {
+            case "any.required":
+              error.message = t("errorRequired");
+              break;
+            case "string.email":
+              error.message = t("errorEmail");
+            default:
+              break;
+          }
+        });
+        return errors;
+      }),
+  });
+
+  const [email, setEmail] = useState("");
+  const [emailError, setEmailError] = useState("");
+
+  const [errorBoxErrors, setErrorBoxErrors] = useState([]);
+  const [errorBoxText, setErrorBoxText] = useState("");
+
+  const handlerClearData = (e) => {
+    e.preventDefault();
+    setEmailError("");
```

Comment:
I think this is a copy paste error. The handleClearData should `setEmail("")` and not `setEmailError("")`.

Unless we want to keep both so in this case just add `setEmail("")`

Reply:
You are correct in that I did miss `setEmail("")`. In case there is an error, we want to clear it as well, so I'll keep `setEmailError("")` and add the other.